.github/workflows/format-rules.yaml

```diff
@@ -0,0 +1,25 @@
+name: format-rules
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - release
+  merge:
+    branches:
+      - develop
+      - release
+
+jobs:
+  test-latest:
+    name: format-rules
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "17.3.0"
+      - name: install prettier
+        run: npm install prettier
+      - name: format unformatted rules
+        run: npx prettier --write $(npx prettier --list-different "./**/*.yaml")
```

Comment:
Will this workflow file write changes back to the repository? I think that will require making a commit. Have you tested this on GitHub itself?